code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -516,9 +530,16 @@ var/global/list/damage_icon_parts = list()
 	if(wear_id)
 		wear_id.screen_loc = ui_id	//TODO
 		if(w_uniform && w_uniform:displays_id)
-			overlays_standing[ID_LAYER]	= image("icon" = 'icons/mob/mob.dmi', "icon_state" = "id")
-		else if(wear_id.contained_sprite)
-			overlays_standing[ID_LAYER] = image("icon" = wear_id.icon, "icon_state" = "[wear_id.icon_state]_w")
+			if(wear_id.contained_sprite)
+				var/IDIcon

```

Comment:
Consider instantiating as `var/icon/IDIcon` instead. Granted, difference is non-existent (practically), but it's better form, since we are not expecting it to be anything but a type icon.